### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.servicefabric.models.FabricErrorException
  fullName: azure.servicefabric.models.FabricErrorException
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - inheritance:
          - type: builtins.object
          type: builtins.BaseException
        type: builtins.Exception
      type: msrest.exceptions.ClientException
    type: msrest.exceptions.HttpOperationError
  langs:
  - python
  module: azure.servicefabric.models
  name: FabricErrorException
  source:
    id: FabricErrorException
    path: azure-servicefabric\azure\servicefabric\models\fabric_error_py3.py
    remote:
      branch: master
      path: azure-servicefabric\azure\servicefabric\models\fabric_error_py3.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 41
  summary: 'Server responsed with exception of type: ''FabricError''.

    '
  syntax:
    content: FabricErrorException(deserialize, response, *args)
    parameters:
    - description: 'A deserializer

        '
      id: deserialize
    - description: 'Server response to be deserialized.

        '
      id: response
  type: class
  uid: azure.servicefabric.models.FabricErrorException
references: []
